### PR TITLE
Jumpstart: Project Content

### PR DESCRIPTION
**Pull request by renejeglinsky**
Hi,
I started to add content but haven't had much time. Either way, I decided to create the PR to have it out there so that you can contribute if you find the time.

Wasn't sure about the index.cds file. I found several options to use an index.cds file and was not sure about the differences. Also not sure why we have a [services.cds in the fiori sample](https://github.com/SAP-samples/cloud-cap-samples/blob/main/fiori/app/services.cds) but mention an [index.cds in capire](https://cap.cloud.sap/docs/advanced/fiori#where-to-put-them).

### CODE REVIEW

File: get-started/jumpstart.md [hunk 1]
```diff
@@ -128,11 +128,20 @@ bookshop/        # Your project's root folder
...
 
+### Project Content
 
+::: tip Neighbouring concept
+Using CAP, it's a rule of thumb that related files are placed next to each other. This is prominently demonstrated with sibling `.js` files, next to `.cds` sources, for [implementing services](/node.js/core-services#implementing-services).
+
+:::
+
+#### Using Subfolders
+The `cds build` assumes some default locations for your files. If you want to deviate from those defaults, for example, to organize annotations files or database-specific files in subfolders, you need to add an _index.cds_ file that contains the information where to find the needed files.
```
Comment (swaldmann) [inline get-started/jumpstart.md:144]: For `cds.resolve` I once played around with cap/cds/pull/3707 to optionally auto-resolve deep folders.

The mandatory `index.cds` have introduced some necessity for overhead tooling (e.g. `cds-indexer`), so I wanted to investigate more elegant solutions.

File: get-started/jumpstart.md [hunk 2]
```diff
@@ -128,11 +128,20 @@ bookshop/        # Your project's root folder
 ├─ app/          # UI-related content
 ├─ srv/          # Service-related content
 ├─ db/           # Domain models and database-related content
+├─ i18n/         # Text bundles
```
Comment (danjoa) [inline get-started/jumpstart.md:131]: That is not added by `cds init` hence we should not document that here. 
It is documented in the i18n gude

**Review by LotharBender — APPROVED** [review]

File: get-started/jumpstart.md [hunk 3]
```diff
@@ -132,7 +132,15 @@ bookshop/        # Your project's root folder
...
 
+### Project Content
 
+::: tip Principle of locality
+Using CAP, it's a rule of thumb that related files are placed next to each other. This is prominently demonstrated with sibling `.js` files, next to `.cds` sources, for [implementing services](/node.js/core-services#implementing-services).
+
+:::
+
+#### Using Subfolders
+The `cds build` assumes some default locations for your files. If you want to deviate from those defaults, for example, to organize annotations files or database-specific files in subfolders, you need to add an _index.cds_ file that contains the information where to find the needed files.
```
Comment (chgeo) [inline get-started/jumpstart.md:143]: We shouldn't be overly specific on which tools assumes a structure.

```suggestion
CAP assumes some default locations for your files. If you want to deviate from those defaults, for example, to organize annotations files or database-specific files in subfolders, you need to add an _index.cds_ file that contains the information where to find the needed files.
```

**Review by chgeo — APPROVED** [review]

**Review by LotharBender — APPROVED** [review]

File: get-started/jumpstart.md (hunk 3 above)
Comment (LotharBender) [reply get-started/jumpstart.md]: The term CAP is definitely the better choice here as the corresponding implementation isn't specific to cds build and used across the entire codebase, e.g. cds deploy...

**Pull request merged**